mixxx-metainfo.patch

```diff
@@ -0,0 +1,146 @@
+diff --git a/res/linux/org.mixxx.Mixxx.metainfo.xml b/res/linux/org.mixxx.Mixxx.metainfo.xml
+index 8696f0e529..f9c5b160c4 100644
+--- a/res/linux/org.mixxx.Mixxx.metainfo.xml
++++ b/res/linux/org.mixxx.Mixxx.metainfo.xml
+@@ -4,6 +4,7 @@
+   <launchable type="desktop-id">org.mixxx.Mixxx.desktop</launchable>
+   <metadata_license>CC-BY-SA-3.0</metadata_license>
+   <project_license>GPL-2.0</project_license>
++  <content_rating type="oars-1.1" />
+   <name>Mixxx DJ Software</name>
+   <summary>Everything you need to perform live DJ mixes</summary>
+   <description>
+@@ -2304,29 +2305,7 @@
+    #4556
+    #4554
+    #4555
+-   #4552
+-   #4549
+-   #4788
+-   #4620
+-   #4641
+-   #4649
+-   #4679
+-   #10928
+-   #10927
+-   #11063
+-   #11153
+-   #11180
+-   #11167
+-   #11181
+-   #11266
+-   #11312
+-   #11389
+-   #4393
+-   #12268
+-   #12240
+-   #12056
+-   #12430
+-   #12029
++   ...
```

Comment:
this should just be a PLAIN ENGLISH high level description of the changes. Not bug numbers. This is meant to be read by **users**

Reply:
This is the place for user visible change logs. It's whats displayed on flathub, on GNOME software or KDE discovery, and some developers even use to display it in their application.

Reply:
This is autogenerated upstream from release announcements, which is why this patch is here to meet Flathub's requirements.